content/devs/cli.md

```diff
@@ -28,10 +28,12 @@ sudo cp drone /usr/local/bin
 You will need to provide the Drone command line tools with the location of your Drone installation as well as an access token for authentication. You can retrieve a Drone personal access token from your user profile screen.
 
 ```bash
-export DRONE_SERVER=<http://>
+export DRONE_SERVER=http://myserver.com
```

Comment:
For the sake of consistency between the two (and to prevent copypasta-but-forgot-to-modify errors), can we leave DRONE_SERVER as it was but add the disclaimer below?